src/crimson/osd/osd.cc

```diff
@@ -1402,6 +1403,82 @@ seastar::future<> OSD::send_beacon()
   return monc->send_message(std::move(beacon));
 }
 
+seastar::future<double> OSD::run_bench(int64_t count, int64_t bsize, int64_t osize, int64_t onum) {
+    LOG_PREFIX(OSD::run_bench);
```

Comment:
make check failed with:
```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/crimson/osd/osd.cc:1407:5: error: unused variable 'FNAME' [-Werror,-Wunused-variable]
    LOG_PREFIX(OSD::run_bench);
```
You can add a `DEBUG();` a line after that to fix the error

Reply:
done